client/Components/amenities.jsx

```diff
@@ -0,0 +1,15 @@
+class Amenities extends React.Component {
```

Comment:
Are you planning on changing the state for these?

Reply:
probably. I think setting state might be easier than just working with the props... but I am still 100% sure what my intentions are... lol